Documentation/hacking.md

```diff
@@ -200,17 +227,27 @@ Now, GOTO [saving](#saving-the-set-of-dependencies)
 
 #### Removing an existing dependency
 
-This is the simplest case of all: simply remove all references to a dependency from the source files.
+This is the simplest case of all.
+
+##### If it is a library
+
+Simply remove all references to a dependency from the source files.
+
+Now, GOTO [saving](#saving-the-set-of-dependencies)
+
+##### If it is an application
+
+Simply remove the relevant line from the vendoredApps file.
 
 Now, GOTO [saving](#saving-the-set-of-dependencies)
 
 ### Saving the set of dependencies
 
 Finally, here we are, the magic command, the holy grail, the ultimate conclusion of all `godep` operations.
-Provided you have followed the preceding instructions, regardless of whether you are adding/removing/modifying dependencies, this command will cast the necessary spells to solve all of your dependency worries:
+Provided you have followed the preceding instructions, regardless of whether you are adding/removing/modifying dependencies, this innocuous script will cast the necessary spells to solve all of your dependency worries:
```

Comment:
Will you two please @ me into all your code reviews? Comedy gold at bronze medal prices!